src/main/java/randoop/generation/Bloodhound.java

```diff
@@ -0,0 +1,302 @@
+package randoop.generation;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import randoop.main.GenInputsAbstract;
+import randoop.operation.CallableOperation;
+import randoop.operation.EnumConstant;
+import randoop.operation.FieldGet;
+import randoop.operation.FieldSet;
+import randoop.operation.MethodCall;
+import randoop.operation.TypedOperation;
+import randoop.util.Randomness;
+import randoop.util.SimpleArrayList;
+
+/**
+ * Implements the Bloodhound component, as described by the paper "GRT: Program-Analysis-Guided
+ * Random Testing" by Ma et. al (appears in ASE 2015):
+ * https://people.kth.se/~artho/papers/lei-ase2015.pdf .
+ *
+ * <p>Bloodhound computes a weight for each method under test by taking a weighted combination of
+ * the uncovered branch ratio and the ratio between the number of times the method has been
+ * successfully invoked (to be the last statement of a new regression test) and the maximum number
```

Comment:
Our definition is about the number of times the method was selected for a test, and the test was a regression test.
Another possible definition would be the total number of times the method appears in any regression test.
Both are consistent with the paper.  I think what we have is more likely to be what the authors intended, but it's impossible to know for sure.

Reply:
My above comment is probably worth adding to the documentation.

Reply:
Documentation now includes comment on the two definitions of "successful invocations".